bp/src/Bluepill.m

```diff
@@ -676,6 +676,18 @@ - (BPSimulator *)test_simulator {
     return self.context.runner;
 }
 
+- (BOOL)canRetryError {
```

Comment:
nit: `canRetryOnError`

Reply:
+1

Reply:
Hey, sorry I missed this comment. Updated and squashed.